lib/bundler/plugin/index.rb

```diff
@@ -95,6 +95,14 @@ def installed?(name)
         @plugin_paths[name]
       end
 
+      def installed_plugins
+        @plugin_paths.keys
+      end
+
+      def plugin_commands(plugin)
+        @commands.find_all{|c,n| n == plugins}.map(&:first)
```

Comment:
Wouldn't be great to have a class for plugins, which lists the commands it have instead of using hashes to do this work? I assume hashes are faster, but classes more readable? We can, of course, add a reverse hash for this specific method.